index.html

```diff
@@ -0,0 +1,48 @@
+
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title> CLARENCE PORTFOLIO</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Roboto:wght@400;700;900&display=swap" rel="stylesheet">
+  <script src="https://kit.fontawesome.com/073688fc60.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
+
+  <!-- Navigation -->
+
+  <header>
+    <a clsss="my-logo" href="#">Clarence</a>
+    <nav>
+      <i class="fa-solid fa-bars"></i>
+    </nav>
+  </header>
+
+  <!-- Hero Section -->
+
+  <section class="hero">
+    <div class="home-header">
+      <h1>Hey There.<br>I'm Clarence</h1>
+      <h3>I'm a software developer</h3>
+      <p>i can help you build a product, feature or web site look throughsome of my workand experiece!If you like whst you see and have a project you need coded,dont hesitate to contact me.</p>
+    </div>
+    <ul class="socials">
+      <li><i class="fa-brands fa-github"></i></li>
+      <li><i class="fa-brands fa-linkedin"></i></li>
+      <li><i class="fa-brands fa-angellist"></i></li>
+      <li><i class="fa-brands fa-twitter"></i></li>
+      <li><i class="fa-brands fa-medium"></i></li>
+    </ul>
```

Comment:
You did really well adding the icons in a `ul, li` tag, please it's necessary to also put them in a `a href=` tag as this will let you connect your social networks with the icons, using href and is also semantically correct 😄 

>    
     <ul>
        <li><a href="#"><i class "..."></i></a></li>
    </ul>